docs/data/base/components/switch/UnstyledSwitchIntroduction.tsx

```diff
@@ -87,10 +87,34 @@ export default function UnstyledSwitches() {
 
   return (
     <div>
-      <Switch component={Root} {...label} defaultChecked />
-      <Switch component={Root} {...label} />
-      <Switch component={Root} {...label} defaultChecked disabled />
-      <Switch component={Root} {...label} disabled />
+      <Switch<typeof Root>
```

Comment:
The generic parameter is needed only when you want to use a prop that's available on the root slot. It's not the case here, so we can simplify and remove it. (x other places)

Reply:
addressed